content/millennium/r4/financial/financial-transaction.md

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
Missing the Adjustment example

Reply:
Added https://github.com/cerner/fhir.cerner.com/pull/297/commits/0e8234a13abfe664bfb577f71a8bbc9a6bb9af88#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR164

Reply:
+1